src/photos.cs

```diff
@@ -1152,4 +1194,266 @@ interface PHLivePhotoEditingOption {
 		NSString ShouldRenderAtPlaybackTime { get; }
 	}
 #endif
+
+	[Mac (10,13, onlyOn64 : true)]
+	[NoiOS][NoTV]
+	[BaseType (typeof (PHAssetCollection))]
+	interface PHProject {
+
+		[Export ("projectExtensionData")]
+		NSData ProjectExtensionData { get; }
+
+		[Field ("PHProjectTypeUndefined")]
+		NSString TypeUndefined { get; }
```

Comment:
Looking at the swift definition for this:

```static let undefined: PHProjectType```

I wonder if this should be in a separate PHProjectType class, to be PHProjectType.Undefined instead of PHProject.TypeUndefined.  Seems like something they may add more options to down the line, and could possibly end up as a smart enum